plasma_framework/contracts/mocks/framework/registries/VaultRegistryMock.sol

```diff
@@ -3,15 +3,10 @@ pragma solidity ^0.5.0;
 import "../../../src/framework/registries/VaultRegistry.sol";
 
 contract VaultRegistryMock is VaultRegistry {
-    uint256 public constant QUARANTINE_PERIOD = 0;
-    uint256 public constant INITIAL_IMMUNE_VAULTS = 0;
+    constructor (uint256 _minExitPeriod, uint256 _initialImmuneVaults)
+        VaultRegistry(_minExitPeriod, _initialImmuneVaults) public {}
```

Comment:
```suggestion
        VaultRegistry(_minExitPeriod, _initialImmuneVaults) public {
    }
```
Minor: lets move closing `}` new line indented as `constructor` (visibility)
Since it's my only comment, lets try suggestion